bundles/org.openhab.binding.sensorcommunity/README.md

```diff
@@ -1,6 +1,6 @@
-# LuftdatenInfo Binding
+# SensorCommunity Binding
 
-Binding for the Sensor Community [luftdaten.info](https://luftdaten.info/). The community provides instructions to build sensors on your own and they can be integrated into the database.
+Binding for the Sensor Community [Sensor.Community](https://sensor.community/). The community provides instructions to build sensors on your own and they can be integrated into the database.
```

Comment:
```suggestion
Binding for the Sensor Community [Sensor.Community](https://sensor.community/).
The community provides instructions to build sensors on your own and they can be integrated into the database.
```

Do we want to mention here that it was previously known as LuftdatenInfo?

Reply:
I think we should not mention it. The project wants to get rid if this name.... Anyhow, we will document it in update.lst.

Reply:
OK. My thinking was that a search for "openHAB" and "LuftdatenInfo" would then still lead to this page. The project was renamed, and we are now renaming as well to comply, but that doesn't mean it never existed by that name, and we are not erasing history. 🙂 However, you are right that it will also be possible to connect the dots through the release notes.